WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/EntryPoint/AddProductWithAIActionSheet.swift

```diff
@@ -71,20 +79,32 @@ struct AddProductWithAIActionSheet: View {
                 Divider()
 
                 // Manual option
-                HStack(alignment: .top, spacing: Constants.margin) {
-                    Image(systemName: "plus.circle")
-                        .font(.title3)
-                        .foregroundColor(.secondary)
-                    VStack(alignment: .leading, spacing: Constants.verticalSpacing) {
-                        Text(Localization.manualTitle)
-                            .bodyStyle()
-                        Text(Localization.manualDescription)
-                            .subheadlineStyle()
+                if !isShowingManualOptions {
+                    HStack(alignment: .top, spacing: Constants.margin) {
+                        Image(systemName: "plus.circle")
+                            .font(.title3)
+                            .foregroundColor(.secondary)
+                        VStack(alignment: .leading, spacing: Constants.verticalSpacing) {
+                            Text(Localization.manualTitle)
+                                .bodyStyle()
+                            Text(Localization.manualDescription)
+                                .subheadlineStyle()
+                        }
+                        Spacer()
+                    }
+                    .onTapGesture {
+                        withAnimation {
+                            isShowingManualOptions = true
+                        }
                     }
-                    Spacer()
                 }
-                .onTapGesture {
-                    onManualOption()
+
+                if isShowingManualOptions {
```

Comment:
Nit: you can use `else` here.

Reply:
Right! added in https://github.com/woocommerce/woocommerce-ios/pull/11859/commits/b84fd11df840cb27816b1af3ddafb61645348115